### DISCUSSION TITLE
Thin walls instead of external perimeters issue

### DISCUSSION THREAD

**Discussion by paezBMTG** [supermerill/SuperSlicer#2993] [General]
![image](https://user-images.githubusercontent.com/104785750/181352647-8d45cfd2-a46e-4b95-a58d-bcc5123e1606.png)

**Comment by supermerill** [comment]
doesn't you put a value two time too high in your external perimeter spacing?

I can provide you with advanced support & advices to solve your issue, but it won't be free. You can contact me on [discord](https://github.com/supermerill/SuperSlicer/issues/611#issuecomment-907833287) or [reddit](https://www.reddit.com/user/supermerill/) (or by email, but I won't write it here). I'm often available in 20h00-22h00 CEST.

If you think you have found a bug, please open an issue with your project file.

**Comment by Mattcgarcia** [comment]
After reading this, I suppose I am having a similar issue with trying to print thin walls or perimeters and having some inconsistency. What is the basic behavior of thin wall? For example, I have thin wall on the default min. width value of 33%, so I was guessing that it would come out to be about one third of the external perimeter width. I am relatively new to SuperSlicer and do not quite understand what it means. 
Thank you!
